app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def show
+    # Post
+  end
+end
```

Comment:
* Kindly add the missing post controller action `index` as when i ran `rails routes` the outcome was different from the expected 
![image](https://github.com/KAYEMBAJONATHAN/Blog-app-/assets/52497246/534215a1-9bb2-4580-afa9-6d8291330a56)